config/params/mainnet_config.go

```diff
@@ -115,6 +117,12 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	ReorgParentWeightThreshold:      160,
 	ReorgMaxEpochsSinceFinalization: 2,
 	IntervalsPerSlot:                3,
+	ProposerReorgCutoffBPS:          primitives.BP(1667),
+	AttestationDueBPS:               primitives.BP(3333),
+	AggregrateDueBPS:                primitives.BP(6667),
+
+	SyncMessageDueBPS:  primitives.BP(3333),
+	ContributionDueBPS: primitives.BP(6667),
```

Comment:
```suggestion
	ProposerReorgCutoffBPS:          primitives.BP(1667),
	AttestationDueBPS:               primitives.BP(3333),
	AggregrateDueBPS:                primitives.BP(6667),
	SyncMessageDueBPS:               primitives.BP(3333),
	ContributionDueBPS:              primitives.BP(6667),
```

if the last two don't belong here in `Fork choice algorithm constants`, we should move them to another group.
maybe it's also a good idea to move all of these to a new `Time parameters` group since there is more stuff coming in gloas.